packages/kbn-securitysolution-exception-list-components/src/list_header/menu_items/index.tsx

```diff
@@ -18,9 +18,9 @@ interface MenuItemsProps {
   linkedRules: Rule[];
   canUserEditList?: boolean;
   securityLinkAnchorComponent: React.ElementType; // This property needs to be removed to avoid the Prop Drilling, once we move all the common components from x-pack/security-solution/common
-  onExportList: () => void;
   onDeleteList: () => void;
   onManageRules: () => void;
+  onExportModalOpen: () => void;
```

Comment:
just to understand, do we need to rename the `onExportList` to `onExportModalOpen`?
I believe the `onExportList` is more generic, as the name `onExportModalOpen` implies that usually we show a model when we export, right 🤔?